open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -11,50 +11,74 @@ servers.
 * The environment variable is called `EGERIA_PRESENTATIONSERVER_SERVER_<ui server name>`  when the <ui server name> is the tenants
  serverName.
 * The value of the environment variable is a json payload with 2 named properties
-    * `remoteServerName` the remote server name
-    * `remoteURL` the remote url. 
+    * `remoteServerName` the remote view server name.
+    * `remoteURL` the server root url for the view server. 
 * An example of the environment variable you would specify for a UI tenant with server name `aaa` with a remote server name
 `  cocoView1` and remote URL of `https://localhost:9443 ` is
  ` EGERIA_PRESENTATIONSERVER_SERVER_aaa={"remoteServerName":"cocoView1","remoteURL":"https://localhost:9443"}`
-* To run the presentation server in production, use the presentation server assembly and specify the environment variables as required.   
+* To run the Presentation Server in production, use the Presentation Server assembly and specify the environment variables as required.   
 
 ## For development
  * Add the presentation-server folder into [Visual Studio Code](https://code.visualstudio.com/). It will create a new workspace.
- * You can open a terminal in Visual Studio code to issue the npm commands
+ * You can open a terminal in Visual Studio Code to issue the npm commands
  * Set up your environment by running `npm install`
  * Developers can specify the environment variable(s) in a .env file, by copying [.env_sample](nodejs/.env_sample) to a file
  called .env in the same folder and amend the values as required.
  * Once the .env file is in place, build and start the server using `npm start`. Use other npm parameters as specified in [package.json](nodejs/package.json).  
 
-## FOR NEW DEVELOPMENT (as of 8/2/2020)
- * Currently, the UI is not optimized to run in development mode. Developers must rebuild the static files every time a change is made.
- * Because of this, work is currently underway to remake the client with Create React App. This will be done iteratively, and updates posted here.
- * Old client code will be maintained until the cutover is complete, so developers can run the traditional way if desired.
- * The new client is maintained in the cra-client directory.
- * Set up your environment by running `yarn`.
- * Enter `http://localhost:3000/` to access the new UI in development mode.
- 
 ## Access UI using the browser. 
- * Enter 'https://localhost:8091/<tenant-name>/' on the UI to access the tenanted UI (<tenant-name> is the serverName used by the presentation server). The Ui will prompt for a login; some of the 
+ * Enter 'https://localhost:8091/<tenant-name>/login' on the UI to access the tenanted UI (<tenant-name> is the serverName used by the presentation server). The Ui will prompt for a login; some of the 
  Coco Pharmaceutical personnas have been enabled here - use user 'faith' and password 'admin'. If there is an environment variable :
```

Comment:
It would be better to use faithbroker rather than faith. I think this is more widely used

Reply:
actually see below -- I needed to use garygeeke for dino to work.. though that is demo specific. for now will change faith to faithbroker as I don't know implications for glossary -- and long term a better approach would suggest different users for the different UIs most likely?

Reply:
changed to faithbroker here and left clarification under the demo section. reopen if more is needed